src/tzdata/download.jl

```diff
@@ -1,4 +1,5 @@
 using Dates
+using Downloads
 using TimeZones: DEPS_DIR
```

Comment:
```suggestion
using TimeZones: DEPS_DIR

if VERSION >= v"1.6.0-DEV.923"
    using Downloads: download
else
    using Base: download
end
```

Reply:
https://github.com/JuliaLang/julia/commit/a39c252cdbfed2d717cd453ca0dc51f7e0be48f0

Reply:
> I'm not sure how to get packages to resolve on 1.0.

Ah, more work to do

Reply:
A couple of options:
- Add a fake Downloads to Compat.jl
- `invokelatest(Downloads().download, ...)`
- Remove pre-1.3 support

Reply:
```suggestion
using TimeZones: DEPS_DIR

if VERSION >= v"1.6.0-DEV.923"
    # Use Downloads.jl once TimeZones.jl drops support for Julia versions < 1.3
    download(args...) = invokelatest(Downloads().download, args...)
else
    using Base: download
end
```